CHANGELOG.asciidoc

```diff
@@ -136,6 +136,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Allow TCP helper to support delimiters and graphite module to accept multiple metrics in a single payload. {pull}8278[8278]
 - Added 'died' PID state to process_system metricset on system module{pull}8275[8275]
 - Added `ccr` metricset to Elasticsearch module. {pull}8335[8335]
+- Added support for query params in configuration {pull}8286[8286]
```

Comment:
Oh, the referenced issue here is the bug report, but the tag is for `pull`. You can reference any of them here, or both, but use `pull` for the pull request and `issue` for the issue. Usually the PR is always referenced and sometimes also the issue.